README.md

```diff
@@ -22,6 +22,12 @@ $ npm run dev
 
 If port 8080 is already in use on your machine you must change the port number in `/config/index.js`. Otherwise `npm run dev` will fail.
 
+## What's not Included
+
+* You'll need to configure your web server to add HTTP headers to prevent caching of critical service worker files.
+If you don't do this, you might end up with browsers that never update the cached content.
```

Comment:
It's not (quite) as bad as that.

Can you update the "never" to say that there might be a delay of up to 24 hours? And link to https://stackoverflow.com/questions/38843970/service-worker-javascript-update-frequency-every-24-hours/38854905#38854905